src/main/java/CollectingCharactersDataViaTerminal.java

```diff
@@ -0,0 +1,12 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class CollectingCharactersDataViaTerminal {
```

Comment:
Nazwa klasy brzmi jak czasownik, a ma ona w sobie stan `charactersNamesList` , `charactersWeightsList`. Generalnie powinniśmy nazywać klasy tak, że jeżeli nazwa klasy przypomina rzeczownik to ona ma właśnie stan, czyli jakieś wartości w środku. A jeżeli nazwa klasy przypomina czasownik to powinna mieć ona operacje, które nie opierają się na stanie klasy. Np. Rzeczownik - 

```java 
Person person  = new Person(name, age, money); 
person.assignNewContract();
person.payDepts();
```
Natomiast klasa-czasownik wyglądała by tak
```java
EmailSender emailSender = new EmailSender();
emailSender.sendEmail(email);
```

Reply:
Generalnie rozbiłbym to jak te dane są pobierane (czyli scanner) od tego jak te dane są przetrzymywane (czyli stan), bo w przyszłości możesz zmienić zdanie i pobierać dane w inny sposób. Jak to trzymasz jako jeden klocek to ciężej jest Ci te klocki komponować z innymi klockami.